tests/lib/Creator/DataSourceTestItemCreator.php

```diff
@@ -0,0 +1,228 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzRecommendationClient\Tests\Creator;
+
+use ArrayIterator;
+use EzSystems\EzRecommendationClient\Tests\Stubs\Criteria;
+use EzSystems\EzRecommendationClient\Tests\Stubs\Item;
+use EzSystems\EzRecommendationClient\Tests\Stubs\ItemList;
+use EzSystems\EzRecommendationClient\Tests\Stubs\ItemType;
+use Ibexa\Contracts\Personalization\Criteria\CriteriaInterface;
+use Ibexa\Contracts\Personalization\Value\ItemInterface;
+use Ibexa\Contracts\Personalization\Value\ItemListInterface;
+use Ibexa\Contracts\Personalization\Value\ItemTypeInterface;
+use Traversable;
+
+final class DataSourceTestItemCreator
+{
+    /**
+     * @phpstan-param ?iterable<string, array{
+     *  'item_type_identifier': string,
+     *  'item_type_name': string,
+     *  'languages': array<string>,
```

Comment:
Just FYI, this variant also will work. If you're consciously using `array<string>`, then it's ok.
```suggestion
     *  'languages': string[],
```

Reply:
Yes, I know this variant also will work. I'm consciously using this notation